base_classes/NXlog.nxdl.xml

```diff
@@ -22,57 +22,57 @@
 -->
 <definition
-	name="NXlog" 
-	version="1.1" 
-    type="group" 
+	name="NXlog"
+	version="1.1"
+    type="group"
     extends="NXobject"
 	category="base"
-	xmlns="http://definition.nexusformat.org/nxdl/3.1" 
+	xmlns="http://definition.nexusformat.org/nxdl/3.1"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
     >
     <doc>
 		Information recorded as a function of time.
-		
+
 		Description of information that is recorded against
 		time. There are two common use cases for this:
 
 		- When logging data such as temperature during a run
 		- When data is taken in streaming mode data acquisition,
 		  i.e. just timestamp, value pairs are stored and
-		  correlated later in data reduction with other data, 
+		  correlated later in data reduction with other data,
 
-		
-		It both cases NXlog contains 
+
+		It both cases NXlog contains
 		the logged or streamed  values and the times at which they were measured as elapsed time since a starting
 		time recorded in ISO8601 format. The time units are
 		specified in the units attribute. An optional scaling attribute
-		can be used to accomodate non standard clocks. 
+		can be used to accomodate non standard clocks.
+
 
-		
 		This method of storing logged data helps to distinguish
 		instances in which a variable is a dimension scale of the data, in which case it is stored
-		in an :ref:`NXdata` group, and instances in which it is logged during the 
+		in an :ref:`NXdata` group, and instances in which it is logged during the
 		run, when it should be stored in an :ref:`NXlog` group.
 
 		In order to make random access to timestamped data faster there is an optional array pair of
 		``cue_timestamp_zero`` and ``cue_index``. The ``cue_timestamp_zero`` will
 		contain coarser timestamps than in the time array, say
 		every five minutes. The ``cue_index`` will then contain the
 		index into the time,value pair of arrays for that
-		coarser ``cue_timestamp_zero``. 
+		coarser ``cue_timestamp_zero``.
 
     </doc>
 	<field name="time" type="NX_FLOAT" units="NX_TIME">
 		<doc>
-			Time of logged entry. The times are relative to the "start" attribute 
+			Time of logged entry. The times are relative to the "start" attribute
 			and in the units specified in the "units"
 			attribute. Please note that absolute
 			timestamps under unix are relative to ``1970-01-01T:00:00``.
 
-			The scaling_factor, when present, has to be applied to the time values in order 
-			to arrive at the units specified in the units attribute. The scaling_factor allows 
-			for arbitrary time units such as ticks of some hardware clock.  
+			The scaling_factor, when present, has to be applied to the time values in order
+			to arrive at the units specified in the units attribute. The scaling_factor allows
+			for arbitrary time units such as ticks of some hardware clock.
 		</doc>
 		<attribute name="start" type="NX_DATE_TIME" />
```

Comment:
add doc similar to ``start`` below?

Reply:
As a forward reference? I wouldn't.

Reply:
not as a forward reference (as if to the *same* documention text)

Suggesting that we add a ``doc`` element that describes *this start attribute.  Or is it too obvious?

Reply:
My hope is that it's too obvious. But I could be convinced otherwise.